src/components/ApplicationModal/ModalContact/modalContact.scss

```diff
@@ -0,0 +1,69 @@
+.info-display {
```

Comment:
Is this the same styling as in `modalBasicInfo.scss`? If yes, can we just reuse one?

Reply:
Yep! Just updated on the branch - think Wen will have to pull to actually update the PR

Reply:
It is fine to reuse the styling here for now, but I generally would advise against it in case we want to expand the styling later on. It would be nice to keep many individual styling decisions separate from one another.